README.md

```diff
@@ -34,7 +34,7 @@ import pystac
 import stac_asset
 
 href = "https://raw.githubusercontent.com/radiantearth/stac-spec/master/examples/simple-item.json"
-item = pystac.from_file(href)
+item = pystac.Item.from_file(href)
```

Comment:
```suggestion
item = pystac.read_file(href)
```

I always get screwed up by the difference between `read_file` and `from_file`. I'd prefer to use the top-level function.